parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -245,7 +245,7 @@ if [[ -n ${MASTER_NODE} ]]; then
   time_metric "EnsureK8sControlPlane" ensureK8sControlPlane
   {{- if HasClusterInitComponent}}
   if [[ $NODE_INDEX == 0 ]]; then
-    retrycmd 120 5 30 $KUBECTL apply -f /opt/azure/containers/cluster-init.yaml || exit {{GetCSEErrorCode "ERR_CLUSTER_INIT_FAIL"}}
+    retrycmd 120 5 30 $KUBECTL apply -f /opt/azure/containers/init || exit {{GetCSEErrorCode "ERR_CLUSTER_INIT_FAIL"}}
```

Comment:
should this be `apply -f /opt/azure/containers/cluster-init.yaml -f /opt/azure/containers/init` instead?